### PR TITLE
Benchmarks/getaddress.io

### PR DESCRIPTION
**Pull request by g-m-taylor**
#### Tests
Does this PR include tests for the changes introduced?
- [] Yes
- [x] No, I found it difficult to test    - can't set API key in circleCI
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/test/unit/utils/getAddressAPI.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const expect = require('chai').expect;
+
+const getAddressAPI = require('../../../utils/getAddressAPI');
+
+describe('fileLock util', () => {
+  it("should return the correct data from API", async () => {
+    const data = await getAddressAPI.getAddressAPI.getPostcodeData("M46FW");
+    expect(data.addresses[0].district).to.equal("Manchester");
+  });
```
Comment (aaron-russell) [inline server/test/unit/utils/getAddressAPI.spec.js:9]: Is this going to work on Circle without an API key?

File: server/utils/getAddressAPI.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        const postcodeData = await axios.get(`https://api.getAddress.io/find/${postcode.replace(/ /g, '')}?api-key=${config.get('getAddress.apikey')}&expand=true`);
+        return postcodeData.data;
+      } catch (err) {
+        console.error(err);
+        return false;
+      }
+    }
+  };
+module.exports.getAddressAPI = getAddressAPI;
+module.exports.getAddressAPI.getPostcodeData = getAddressAPI.getPostcodeData;
```
Comment (aaron-russell) [inline server/utils/getAddressAPI.js:17]: Do you need to export the child if you're exporting the parent too?

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: server/test/unit/utils/getAddressAPI.spec.js (hunk 1 above)
Comment (g-m-taylor) [reply server/test/unit/utils/getAddressAPI.spec.js:9]: Thats what I was testing :P Turns out: nope

File: server/utils/getAddressAPI.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+var config = require('../../server/config/config');
```
Comment (aaron-russell) [inline server/utils/getAddressAPI.js:1]: Why is this a var?

**Review by aaron-russell — CHANGES_REQUESTED** [review]

**Review by aaron-russell — APPROVED** [review]

**Pull request merged**